lib/eclipse/README.jdt-upgrade

```diff
@@ -11,3 +11,5 @@ failing).
 Also move the jdtCompilerAdapter.jar out of the jdt jar file and rename it to reflect the
 version. E.g. if the jdt jar file was named org.eclipse.jdt.core-3.8.2.v20120814-155456.jar,
 rename the jdtCompilerAdapter.jar to jdtCompilerAdapter-3.8.2.v20120814-155456.jar
```

Comment:
Also remove these earlier lines, since they no long apply? Maybe replace with a note about downloading right from maven central, and the "gcj vs batch" details for dependencies (i.e. did you download gcj and then rename? etc)